src/lib/wagmi.ts

```diff
@@ -89,6 +95,61 @@ const WAGMI_SUPPORTED_CHAINS: Chain[] = [
   kava,
 ];
 
+enum ChainId {
+  ETH_MAINNET = '1',
+  ETH_SEPOLIA = '11155111',
+  POLYGON_MAINNET = '137',
+  POLYGON_MUMBAI = '80002',
+  OPT_MAINNET = '10',
+  OPT_SEPOLIA = '11155420',
+  ARB_MAINNET = '42161',
+  ARB_SEPOLIA = '421614',
+  BASE_MAINNET = '8453',
+  BASE_SEPOLIA = '84532',
+}
+
+const getAlchemyRPCUrls = (chainId: ChainId, apiKey: string) => {
```

Comment:
Wagmi v2 deprecated providers, which where pretty convenient. Instead we have to manually provide the RPCs. Here we determine the alchemy RPC based on the chainId